spec/controllers/annotation_categories_controller_spec.rb

```diff
@@ -683,11 +700,11 @@
         test_category_name = 'test_category'
         test_content = 'c6conley'
         found_cat = false
-        AnnotationCategory.all.each do |ac|
+        AnnotationCategory.find_each do |ac|
```

Comment:
This change isn't related to the purpose of this PR and should not be included. Same as the change on line 707.

Reply:
Hi @david-yz-liu - thanks for the feedback! The reason for these changes was because they was enforced by `rubocop` in the pre-commit hook. I'm using `rubocop` v1.58.0, `rubocop-rails` v2.19.1 and `rubocop-performance` v1.17.1 as found in `.pre-commit-config.yaml`. Are there any steps you can suggest to take from here so it doesn't complain about this?

![image](https://github.com/MarkUsProject/Markus/assets/56097527/ee78018d-b81e-4e39-b4f1-1374a07948a4)

Reply:
I'm not sure what's happening given that the pre-commit hooks have not flagged this existing code earlier.

1. Revert the change
2. run the pre-commit hooks
3. if there is an error, show me on slack